### PR TITLE
[Console] Updated section 'Building a single Command Application' with the new 'SingleCommandApplication'

### LINKED ISSUE #12903: [Console] Add SingleCommandApplication to ease creation of Single Comma…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#34819
| PR author(s) | @lyrixx
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by lyrixx**
fixes #12903

### CODE REVIEW

File: components/console/single_command_tool.rst [hunk 1]
```diff
@@ -5,34 +5,36 @@ Building a single Command Application
...
-        ->getApplication()
-        ->setDefaultCommand('echo', true) // Single command application
+    use Symfony\Component\Console\SingleCommandApplication;
+
+    (new SingleCommandApplication())
+        ->setName('My Super Command') // Optional
+        ->setVersion('1.0.0') // Optional
+        ->addArgument('foo', InputArgument::OPTIONAL, 'The directory')
+        ->addOption('bar', null, InputOption::VALUE_REQUIRED)
+        ->setCode(function(InputInterface $input, OutputInterface $output) {
```
Comment (HeahDude) [inline components/console/single_command_tool.rst]: Missing a space after `function`

**Review by HeahDude — APPROVED** [review]
Looks good, with a minor comment

File: components/console/single_command_tool.rst (hunk 1 above)
Comment (lyrixx) [reply components/console/single_command_tool.rst]: thanks, fixed :+1:

**Comment by OskarStark** [comment]
Thanks Grégoire.

**Pull request merged**